File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/transaction/ProcessorCategory.java
package com.example.cf_sdk.logic.changebanksdk.model.transaction;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 *
 * Processor categories.
 */

public enum ProcessorCategory implements Serializable {
    UNKNOWN,

    @SerializedName("CREDIT")
    CREDIT,

    @SerializedName("DEBIT")
    DEBIT
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/transaction/ProcessorCategory.kt
package com.example.cf_sdk.changebankapi.model.transaction

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 *
 * Processor categories.
 */
enum class ProcessorCategory : Serializable {
    UNKNOWN, @SerializedName("CREDIT")
    CREDIT, @SerializedName("DEBIT")
    DEBIT
}